irc/websocket.go

```diff
@@ -0,0 +1,56 @@
+package irc
+
+import (
+	"github.com/gorilla/websocket"
+	"net"
+	"net/http"
+	"time"
+)
+
+var upgrader = websocket.Upgrader{
+	ReadBufferSize:  1024,
+	WriteBufferSize: 1024,
+	CheckOrigin:     func(r *http.Request) bool { return true },
```

Comment:
I think this merits some sort of comment. Why is any origin okay? What is CheckOrigin exactly? :)